functions/heli/fn_initHeli.sqf

```diff
@@ -0,0 +1,142 @@
+		[_handle, 0, ["ACE_MainActions"], _action] call ace_interact_menu_fnc_addActionToObject;
+	};
+} forEach _types;
+
+// Hangar cleaning action
+_modifier = {
+    params ["_t", "_p", "_args", "_actionData"];
+
+	// Retrieving action parameters
+	_args params ["_hangar"];
+
+	// Get the nearest chopper
+	_heli = [_hangar] call GSRI_fnc_getHeli;
+
+	// Proper formatting of displayName and action parameters
+    _actionData set [1, format ["Supprimer %1", [_heli] call GSRI_fnc_minifyName]];
+};
+_condition = {
+	params ["_t", "_p", "_args"];
+	_args params ["_hangar"];
+	!isNull ([_hangar] call GSRI_fnc_getHeli)
+};
+_statement = {
+	params ["_t", "_p", "_args"];
+	_args params ["_hangar"];
+	_heli = ([_hangar] call GSRI_fnc_getHeli);
+	["HeliDelete", [getText (configFile >> "CfgVehicles" >> typeOf _heli >> "displayName")]] call BIS_fnc_showNotification;
+	deleteVehicle _heli;
+};
+_action = ["actionClear","Supprimer","",_statement,_condition,{},[_hangar],"",2,[false, false, false, false, false], _modifier] call ace_interact_menu_fnc_createAction;
+[_handle, 0, ["ACE_MainActions"], _action] call ace_interact_menu_fnc_addActionToObject;
+
+// FRIES mounting action
+_condition = {
+	params ["_t", "_p", "_args"];
+	_args params ["_hangar"];
+	_heli = [_hangar] call GSRI_fnc_getHeli;
+	(isNumber (configFile >> "CfgVehicles" >> typeOf _heli >> "ace_fastroping_enabled") && isNull (_heli getVariable ["ace_fastroping_FRIES", objNull]));
+};
+_statement = {
+	params ["_t", "_p", "_args"];
+	_args params ["_hangar"];
+	_heli = [_hangar] call GSRI_fnc_getHeli;
+	["HeliFRIES", [getText (configFile >> "CfgVehicles" >> typeOf _heli >> "displayName")]] call BIS_fnc_showNotification;
+	[_heli] remoteExecCall ["ace_fastroping_fnc_equipFRIES", 2];
+};
+_action = ["actionFRIES","Equiper FRIES","",_statement,_condition,{},[_hangar]] call ace_interact_menu_fnc_createAction;
+[_handle, 0, ["ACE_MainActions"], _action] call ace_interact_menu_fnc_addActionToObject;
+
+// Place helicopter on rear deck
+// TODO : replace by Lesh's tow mod
+_condition = {
+	params["_t", "_p", "_args"];
+	_args params ["_hangar", "_deck"];
+	// Return true if there is a heli in hangar and if there is no heli on deck
+	!(isNull ([_hangar] call GSRI_fnc_getHeli)) and (isNull ([_deck] call GSRI_fnc_getHeli))
+};
+_statement = {
+	params["_t", "_p", "_args"];
+	_args params ["_hangar", "_deck"];
+	_heli = [_hangar] call GSRI_fnc_getHeli;
+	["HeliMoved", [getText (configFile >> "CfgVehicles" >> typeOf _heli >> "displayName")]] call BIS_fnc_showNotification;
+	_heli setPosASL getPosASL _deck;
+};
+_action = ["actionDeck","Placer l'appareil sur le pont d'envol","",_statement,_condition,{},[_hangar, _deck]] call ace_interact_menu_fnc_createAction;
+[_handle, 0, ["ACE_MainActions"], _action] call ace_interact_menu_fnc_addActionToObject;
+
+// Place helicopter in hangar
```

Comment:
Comment denotes a good candidate for an external function

Reply:
this is a temporary function and will not be externalized